docs/pages/guides/metadata.mdx

```diff
@@ -18,6 +18,20 @@ To configure the pipeline with the metadata, you have two options: Save them loc
 
 For this option, you can save the three files `locations.json`, `sensors.json`, and `campaigns.json` to the `config/` directory of the pipeline directory. Be sure to include all the files - even if you don't have any campaigns, save an empty list to `campaigns.json`.
 
+Alternatively, you can store these files locally in a location outside of this repository. This is achieved by setting the environment variable `ERP_CONFIG_DIR` to the full path of the alternate location (ERP <- EM27 Retrieval Pipeline). Note that this should be the same directory that contains the `config.json` file. If this environment variable is not set, the default location the pipeline will look for these files is `./config/`. When setting an alternate config directory, leave `./config/config.template.json` for tests (this file is version-controlled).
+
```

Comment:
> When setting an alternate config directory, leave `./config/config.template.json` for tests (this file is version-controlled).

I don't think this is true! The tests should not be affected, hence, the `config.template.json` should ne be required in other config directories.

Reply:
There are two function calls that look for `config/config.template.json`: 
- `tests/repository/test_config_template.py` line 9
- `tests/fixtures.py` line 54
What I am trying to say here is that if the user specifies another config directory, they should leave the folder and file `config/config.template.json`. These tests are specifically testing this template file I suppose? If the user deletes this file these functions wont find it (I don't know if they are called during the different test options) and also the git working tree will notice it was deleted, so that might be a bit annoying. I guess what I'm trying to say here is that the user can just leave the repository as it is and make a new folder somewhere for the metadata. For example, in the case they would drag and drop the config folder somewhere else vs just creating a new folder and adding the files that would happen. What do you think?

Reply:
The wording has now been modified slightly.